uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -386,7 +397,7 @@ define([
         //if set indicates self leaverequest is either being managed or edited
         if (this.directiveOptions.leaveRequest) {
           //get a clone so self it is not the same reference as passed from callee
-          attributes = _.cloneDeep(this.directiveOptions.leaveRequest.attributes());
+          attributes = this.directiveOptions.leaveRequest.attributes();
```

Comment:
Because uploader gets stuck while calling uploadAll and the promise is never resolved if its cloneDeep'ed or angular.copy'ed. Also, its not affecting the report view so reverted to direct copy of attributes